tool/jt.rb

```diff
@@ -1114,6 +1114,17 @@ def gem(*args)
     ruby '-S', 'gem', *args
   end
 
+  def ensure_gem_installed(gem_name, version = '', lib = gem_name)
+    # Use `gem_name.upcase` to match what rubygems.rb does.
+    env = {"GEM_REQUIREMENT_#{gem_name.upcase}" => "= #{version}"}
+
+    unless run_ruby(env, "-r#{lib}", '-e1', {:continue_on_failure => true, [:err, :out] => :close})
+      run_ruby({}, '-S', 'gem', 'install', gem_name, '-v', version)
+    end
+
+    env
+  end
```

Comment:
This feels too brittle. I would much rather just let the program fail if the necessary gem is not installed, and let the user install it.
FWIW, there is `Gem.install` so it's possible to install gems without needed extra subprocesses, e.g., in tool/profile_mri.rb